backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/IteratorHandler.scala

```diff
@@ -201,7 +202,7 @@ class IteratorHandler extends IteratorApi with Logging {
 
     val beforeBuild = System.nanoTime()
 
-    val transKernel = NativePlanEvaluator.create()
+    val transKernel = NativePlanEvaluator.create(ExecutionCtxs.contextInstance())
```

Comment:
The change doesn't seem to make real difference?

Reply:
Since the current code `NativePlanEvaluator.create()` is just a shortcut to `NativePlanEvaluator.forCtx(ExecutionCtxs.contextInstance())`